Add CPR_SKIP_INSTALL option (#1305)
libcpr/cpr: CMakeLists.txt

message(STATUS "C++ Requests CMake Options")
message(STATUS "=======================================================")

> cpr_option(CPR_SKIP_INSTALL "Set to ON to use as bundled dependency" OFF)

Reviewer: Negative options are always harder to get your head around. How about:

```suggestion
cpr_option(CPR_ENABLE_INSTALL "Set to ON to enable all cpr install targets." ON)
```

Author: Good
